From 7a00bad6112f600caa567c4b15be34173b6cc3ce Mon Sep 17 00:00:00 2001
From: mdraw <martin.drawitsch@gmail.com>
Date: Wed, 5 May 2021 15:50:22 +0200
Subject: [PATCH] Do not try to remove tmpdir on Windows

This line is not necessary in the conda-forge build env and it fails
with
  PermissionError: [WinError 32] The process cannot access the file because
  it is being used by another process
---
 tensorboard/data/server/pip_package/build.py | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tensorboard/data/server/pip_package/build.py b/tensorboard/data/server/pip_package/build.py
index b93b06eb..88322dcd 100644
--- a/tensorboard/data/server/pip_package/build.py
+++ b/tensorboard/data/server/pip_package/build.py
@@ -100,7 +100,6 @@ def main():
     wheel = wheels[0]
     outwheel = os.path.join(args.out_dir, os.path.basename(wheel))
     shutil.copyfile(wheel, outwheel)
-    shutil.rmtree(tmpdir)
 
     print(outwheel)
 
-- 
2.31.1

